bundles/org.openhab.binding.tellstick/pom.xml

```diff
@@ -14,7 +14,8 @@
   <name>openHAB Add-ons :: Bundles :: Tellstick Binding</name>
 
   <properties>
-    <bnd.importpackage>com.barchart.udt*;resolution:=optional,com.google.protobuf;resolution:=optional,com.jcraft.jzlib;resolution:=optional,com.luckycatlabs.sunrisesunset*;resolution:=optional,io.netty.handler.codec.dns;resolution:=optional,javax.activation;resolution:=optional,org.apache.commons.cli;resolution:=optional,org.apache.tomcat*;resolution:=optional,org.bouncycastle*;resolution:=optional,org.eclipse.jetty*;resolution:=optional,org.eclipse.swt*;resolution:=optional,org.jboss.marshalling*;resolution:=optional,sun.net.dns*;resolution:=optional,sun.security*;resolution:=optional</bnd.importpackage>
+    <bnd.importpackage>!com.luckycatlabs.*,!com.jcraft.jzlib.*,!org.apache.commons.cli.*,!org.eclipse.swt.*</bnd.importpackage>
+    <dep.noembedding>netty-transport-native-unix-common,netty-common,netty-transport,netty-transport-native-epoll,netty-buffer,netty-resolver,netty-codec,netty-codec-http,netty-handler</dep.noembedding>
```

Comment:
@J-N-K What is the reason netty is not included as `provided` here? But as `compile`? Since netty is provided via `tp-netty` feature, I would expect that including it via `provided` would be enough?

Reply:
IIRC this is needed for bnd to resolve the binding-bundle in the demo-app. Scope `provided` means it is always available and that was (at least in the past) not true for the netty-bundles (only if `<feature>openhab.tp-netty</feature>` is installed). Since bnd doesn't care about karaf features it was not able to resolve the binding when the scope is not `compile`.